css/templates/e-commerce/page.css

```diff
@@ -328,6 +332,14 @@ amp-selector [option][selected] {
   text-indent: -var(--space-2);
 }
 
+.ampstart-headerbar-fixed-link-wrapper {
+  width: 25px;
+}
+
+.ampstart-headerbar-fixed-link {
+  margin-right: 0;
```

Comment:
yes but mr2 is hardcoded in the template and adding mr0 is classes wouldn't have a higher priority.